GitHubReactions.ps1

```diff
@@ -129,13 +166,31 @@ filter Get-GitHubReaction
         'RepositoryName' = (Get-PiiSafeString -PlainText $RepositoryName)
     }
 
-    $uriFragment = "/repos/$OwnerName/$RepositoryName/issues/$IssueNumber/reactions"
+    $splatForAddedProperties = @{
+        OwnerName = $OwnerName
+        Repository = $RepositoryName
+    }
+
+    if ($Issue)
+    {
+        $issueOrPullRequest = $Issue
+        $descriptionText = "Issue"
```

Comment:
Just from a code readability/maintainability standpoint, having `descriptionText` and `description` is confusing.
Can you use something like `targetObjectTypeName` or something instead?
and instead of `issueOrPullRequest`, use `targetObjectNumber`?
Here and elsewhere.